components/frontend/src/dashboard/MetricSummaryCard.js

```diff
@@ -90,8 +89,8 @@ export function MetricSummaryCard({ header, onClick, selected, summary, maxY })
         width: Math.max(bbWidth, 1), // Prevent "Failed prop type: Invalid prop range supplied to VictoryBar"
     }
     return (
-        <FilterCard onClick={onClick} selected={selected}>
-            <div ref={ref} style={{ height: "72%" }} aria-label={ariaChartLabel(summary)}>
+        <DashboardCard onClick={onClick} selected={selected} title={header}>
+            <div ref={ref} aria-label={ariaChartLabel(summary)} style={{ height: "72%" }}>
```

Comment:
Where does the 72% height come from, and can it be moved to css?

Reply:
Because the donut charts are round and the cards are higher than wide, the chart needs to have a height that is equal to the width of the card. I guess (can't remember) the 72% is the default ratio between the height and width of the card. I guess it would be better to get the width of the card and use that for the height of the donut chart, but I'm not sure how to do that...